packages/expo-modules-core/common/cpp/EventEmitter.cpp

```diff
@@ -71,6 +71,11 @@ NativeState::Shared NativeState::get(jsi::Runtime &runtime, jsi::Object &object,
   return nullptr;
 }
 
+#pragma mark - SubscriptionNativeState
+
+SubscriptionNativeState::SubscriptionNativeState(jsi::Object &emitter, jsi::Function &listener)
+  : jsi::NativeState(), emitter(std::move(emitter)), listener(std::move(listener)) {}
```

Comment:
using move semantics for referenced `jsi::Object &emitter` looks suspicious. is that intentional?